test/index.test.js

```diff
@@ -59,56 +57,66 @@ describe('Ramen', function() {
   });
 
   describe('loading', function() {
-    it('should load', function(done){
+    it('should load', function(done) {
       analytics.load(ramen, done);
     });
   });
 
   describe('after loading', function() {
-    beforeEach(function(done){
+    beforeEach(function(done) {
       analytics.once('ready', done);
       analytics.initialize();
     });
 
-    describe('#group', function() {
+    describe('#page', function() {
       beforeEach(function() {
         analytics.stub(window.Ramen, 'go');
       });
 
-      it('should not call Ramen.go if ramenSettings is blank', function() {
-        analytics.group('id');
-        analytics.assert(!window.ramenSettings);
+      it('should not call Ramen.go', function() {
+        analytics.page();
         analytics.didNotCall(window.Ramen.go);
       });
 
-      it('should set company ID & call Ramen.go() if ramenSettings exists', function() {
-        window.ramenSettings = {
-          organization_id: '1234567890',
-          user: { email: 'ryan@ramen.is', name: 'Ryan', id: '1234' }
-        };
+      it('should call Ramen.go twice', function() {
+        analytics.identify('12345', { email: 'ryan@ramen.is' });
+        analytics.page();
+        analytics.calledTwice(window.Ramen.go);
```

Comment:
Hey @ndhoule,

Apologies for the delay here. Got caught up in a bunch of other stuff. I'm really banging my head against a wall here. There are some failing tests here right now. I call `identify` and on subsequent calls to `page` or `group`, `analytics.user().email` is blank. I would think I'd be able to access that. Can you provide some guidance for how to properly write these tests to stub out the user's email/id?

Thanks!